controls/datagrid/columns/reordering.md

```diff
@@ -31,7 +33,32 @@ Define the ViewModel:
 
 The result on mobile:
 
-![DataGrid Reordering Phone](../images/datagrid-reordering-ios.gif)
+![DataGrid Reordering Phone](../images/datagrid-reordering-android.gif)
+
+### ColumnReorderIndicatorTemplate
+
+The following example shows how to define the `ColumnReorderIndicatorTemplate` in XAML:
+
+Define the DataTemplate for the Indicator in the Resources of the page:
+```XAML
+<ContentView.Resources>
+    <DataTemplate x:Key="reorderingIndicator">
+        <Label Text="Place here" TextColor="Red"/>
+    </DataTemplate>
+</ContentView.Resources>
+```
+
+Define the property in the DataGrid:
+```XAML
+<telerik:RadDataGrid x:Name="dataGrid"
+                     Grid.Row="1"
+                     CanUserReorderColumns="{Binding IsReorderingEnabled}"
+                     ItemsSource="{Binding Data}"/>   
+```
+
+The result on mobile:
+
+![DataGrid Reordering Indicator Phone](../images/datagrid-reordering-indicator.png)
```

Comment:
images are two big. there is a border abound the right part of the image. 
At all the images do not look good